### PR TITLE
Takeaway-challenge

### PR DESCRIPTION
**Pull request by mmar8**
Not finished, but did as much as I could.

### CODE REVIEW

File: lib/takeaway.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        @menu = { 'burger': 5,
+                  'pizza': 6,
+                  'lasagna': 4,
+                  'ice_cream': 2
+                }
+        @order = []
+    end 
+
+    def menu_print
+        @menu.each { |key, value| puts "#{key}: £#{value}" }
```
Comment (steve-f-kahn) [inline lib/takeaway.rb]: Is there a way you could output the price in correct format £6.00. You could try '%.2f' % value

File: lib/takeaway.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    end 
+
+    def add_item(item_from_menu)
+        @order << item_from_menu
```
Comment (steve-f-kahn) [inline lib/takeaway.rb]: The user wants to be able to add multiple of the same item at the same time. That is not possible at the moment. How could you do this? Anything they pass will go onto the order. It doesn't matter what it is. This will break your code later when they try to total the order as they are not inputing a hash with an integer assigned as the value. Given more time it is important to test for edge cases.

File: lib/takeaway.rb [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    end
+
+    def  price(item)
+        @menu.fetch(item).to_f
+     end 
+    
+     def total 
+        @total = []
+        @order.each { |hash| @total << hash}
+
```
Comment (steve-f-kahn) [inline lib/takeaway.rb]: def total is missing an end, your code can't be working. The 'total' method is not tested at all. This is a reason why we write test before code because if we run out of time our production code (the code we function test) still works.

File: lib/takeaway.rb [hunk 4]
```diff
@@ -0,0 +1,33 @@
+class Restaurant
+    attr_reader :menu, :order
+
+    def initialize
+        @menu = { 'burger': 5,
+                  'pizza': 6,
+                  'lasagna': 4,
+                  'ice_cream': 2
+                }
+        @order = []
```
Comment (steve-f-kahn) [inline lib/takeaway.rb]: Are you going to create a new restaurant for each order? I suggest a new class is needed for the customer.
